### ISSUE TITLE
[Image]: QA: Magnifier button(left bottom) is not visible on the image in 200 or 400% zoom.

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8960]
### Description

Page Element:
Magnifier button

Issue Description:
Magnifier button(left bottom) is not visible on the image when the page is zoomed to 200 or 400%.

Expected:
All the interactive elements in the page should be visible clearly and accessible in 200% and 400% zoom level as well.

User Impact:
When the elements are not available in 200% or 400% zoom, user with low vision who uses zoom feature will not be aware of the element in the page.

### Component(s) impacted

Image

### Browser

Firefox

### Carbon for IBM.com version

v1.35.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
